### DISCUSSION TITLE
Correction or extension in the documentation for pairing Xiaomi Button WXKG01LM

### DISCUSSION THREAD

**Discussion by Marcossi** [Koenkk/zigbee2mqtt#13673] [Devices]
First of all I am sorry for my bad English

Today I replaced the zigbee adapter and had to re-pair two buttons model Xiaomi WXKG01LM. I had a lot of trouble following the pairing steps in the documentation (the buttons were fine because they worked for years) and I think I discovered a problem.

Actually the documentation says:

> Press and hold the reset button on the device for +- 5 seconds (until the blue light starts blinking). After this the device will automatically join. If this doesn't work, try with a single short button press

That "If this doesn't work, try..." made me suspect there were problems, and indeed there are many people on forums saying it doesn't work. I ended up finding the solution on the forums, and it worked on the first try with no problems:

> Press the link button for a few seconds and the led will flash. After this first phase, you have to make quick clicks on the button (normal clicks on the device, not on the link button) until the LED flashes again. After this second flashing phase the device will be paired

From what I understood this "two-stage" pairing is common on xiaomi devices.

If the documentation could be updated or a small annex could be added as a "second alternative" it would be great for the community

Thanks

**Comment by Koenkk** [comment]
You can easily edit the page by clicking the " Help to make the docu better and edit this page on Github ✌ " on the bottom of https://www.zigbee2mqtt.io/devices/WXKG01LM.html

**Reply by Marcossi** [reply]
oh! sorry. I didn't see the option. I'll use it right away! thanks
